astropy/units/astrophys.py

```diff
@@ -16,6 +16,8 @@
 
 import numpy as _np  # noqa: E402
 
+__all__: list[str] = []
```

Comment:
Maybe add a comment `#  Units are added at the end`

Reply:
Done!